README.md

```diff
@@ -1,91 +1,90 @@
+
+- A reincarnation of [this old project](https://github.com/jacoobes/sern_handler)
 
 ## Installation
 
 ```sh
-npm install sern-handler
+npm install @sern/handler
 ```
+
 ```sh
-yarn add sern-handler
+yarn add @sern/handler
 ```
+
 ```sh
-pnpm add sern-handler
+pnpm add @sern/handler
 ```
 
 ## Basic Usage
 
-#### ` index.js `
+#### ` index.js (CommonJS)`
+
 ```js
-import { Client, Intents } from 'discord.js';
-import { Sern } from 'sern-handler';
-import { prefix, token } from '../src/secrets.json';
+const { Client, Intents } = require('discord.js');
+const { Sern } = require('sern-handler');
+const { prefix, token } = require('./config.json');
 
 const client = new Client({
-    intents: [
-        Intents.FLAGS.GUILDS,
-        Intents.FLAGS.GUILD_MESSAGES,
-        Intents.FLAGS.GUILD_MEMBERS
-    ]
+  intents: [
+    Intents.FLAGS.GUILDS, 
+    Intents.FLAGS.GUILD_MESSAGES,
+    Intents.FLAGS.GUILD_MEMBERS
+   ],
 });
 
 new Sern.Handler({
-    client,   
-    prefix,   
-    commands : 'dist/commands', 
-    privateServers : [           
-        {
-            test : true,
-            id: 'server-id'
-        }
-    ],
-    init: async (handler) => {
-        // Optional function to initialize anything else on bot startup
+  client,
+  prefix,
+  commands: 'src/commands',
+  privateServers: [
+    {
+      test: true,
+      id: 'server-id',
     },
+  ],
+  init: async (handler) => {
+    // Optional function to initialize anything else on bot startup
+  },
 });
 
-
 client.login(token);
 ```
 
-#### ` ping.js `
+#### ` ping.js (CommonJS)`
+
 ```js
-import { Sern, Types } from 'sern-handler';
-import { Ok } from 'ts-results';
-
-export default  {
-    alias: [],
-    desc : 'A ping pong command',
-    visibility : 'private',
-    test : false,
-    type: Sern.CommandType.SLASH | Sern.CommandType.TEXT,
-    execute : async ({ message, interaction }, args) => 'pong!'
+const { Sern, Types } = require('sern-handler');
+
+module.exports = {
+  alias: [],
+  desc: 'A ping pong command',
+  visibility: 'private',
+  test: false,
+  type: Sern.CommandType.SLASH | Sern.CommandType.TEXT,
+  execute: async ({ message, interaction }, args) => 'pong!',
 };
 ```
 
 See [documentation](https://sern-handler.js.org) for TypeScript examples and more
 
-## Links ![link](https://img.shields.io/badge/Coming-Soon-purple)
+## Links
 
-- [Official Documentation](https://tmp.com)
-- [Example Bot](https://tmp.com)
-- [Discord Server](https://google.com)
+- **[Official Documentation](https://sern-handler.js.org)**
+- **[Public Templates](https://github.com/sern-handler/templates)**
+- **[Discord Server](https://discord.com/invite/Yvb7DnqjXX)**
 
 ## Contribute
 
--   Pull up on [issues](https://github.com/sern-handler/Sern/issues) and tell us, if there are bugs
--   All kinds of contributions are welcomed!
+- Read our contribution [guidelines](https://github.com/sern-handler/handler) carefully
```

Comment:
I'd suggest commit a `CONTRIBUTING.md` too and link it here

Reply:
Done